src/app/features/current-account/current-account-name.tsx

```diff
@@ -20,9 +20,7 @@ function AccountNameTitle({ children, ...props }: HasChildren & BoxProps) {
 }
 
 const AccountNameSuspense = memo((props: BoxProps) => {
-  const currentAccount = useCurrentStacksAccount();
   const { data: name = 'Account' } = useCurrentAccountDisplayName();
-  if (!currentAccount || typeof currentAccount.index === 'undefined') return null;
```

Comment:
This seems un-necessary as we show a fallback default and always have at least one account to show